Frontend-Development/Job-Boards/Readme.md

```diff
@@ -15,3 +15,4 @@
 - [Remote Frontend Engineer](https://4dayweek.io/remote-jobs/frontend-engineer) - 4/day week
 - [Remote Frontend Jobs](https://www.remotefrontendjobs.com/) - remotefrontendjobs
 - [Junior Frontend Jobs](https://www.jrdevjobs.com/jobs#page=1&query=Frontend+developer) - Jr.Dev jobs
+- [`Peerlist` is a professional community with robust work profiles at its core. For job opportunities and more...](https://peerlist.io/)
```

Comment:
Please add specific job-related searches. Don't add only the platform. Add some filters to only show frontend jobs or backend jobs according to the directory